tests/ImageSharp.Tests/Formats/Jpg/ReferenceImplementationsTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿// ReSharper disable InconsistentNaming
+namespace ImageSharp.Tests.Formats.Jpg
+{
+    using System.Numerics;
+    using ImageSharp.Formats;
+    using ImageSharp.Formats.Jpg;
+
+    using Xunit;
+    using Xunit.Abstractions;
+
+    public class ReferenceImplementationsTests : UtilityTestClassBase
+    {
+        public ReferenceImplementationsTests(ITestOutputHelper output)
+            : base(output)
+        {
+        }
+
+
+        [Theory]
+        [InlineData(42)]
+        [InlineData(1)]
+        [InlineData(2)]
+        public void Idct_FloatingPointReferenceImplementation_IsEquivalentToIntegerImplementation(int seed)
+        {
+            MutableSpan<int> intData = Create8x8RandomIntData(-200, 200, seed);
+            MutableSpan<float> floatSrc = intData.ConvertToFloat32MutableSpan();
+
+            ReferenceImplementations.IntegerReferenceDCT.TransformIDCTInplace(intData);
+            
+            MutableSpan<float> dest = new MutableSpan<float>(64);
+            MutableSpan<float> temp = new MutableSpan<float>(64);
+
+            ReferenceImplementations.iDCT2D_llm(floatSrc, dest, temp);
+
+            for (int i = 0; i < 64; i++)
+            {
+                float expected = intData[i];
+                float actual = dest[i];
+
+                Assert.Equal(expected, actual, new ApproximateFloatComparer(1f));
```

Comment:
Wouldn't it be easier to use the overload specifying the decimal places as we have elsewhere?